composer.lock

```diff
@@ -3547,5 +3555,5 @@
     "platform-overrides": {
         "php": "7.3.0"
     },
-    "plugin-api-version": "1.1.0"
+    "plugin-api-version": "2.0.0"
```

Comment:
This looks suspicious, but since the CI passes, I suppose this doesn't prevent using Composer 1 to install vendor packages like I thought it would.

Reply:
> This looks suspicious

I used Composer 2 to make the update, so it's expected.

> I suppose this doesn't prevent using Composer 1 to install vendor packages like I thought it would.

It shouldn't. It just guarantees that the dependencies were installed with Composer 2, and its plugin API is available for packages (e.g. `Composer\PackageVersions`). And it's only for CI anyways.